### PR TITLE
added Swaychain to alchemy

### PR DESCRIPTION
**Pull request by javiermanzano**
Adding Swaychain NFT

### CODE REVIEW

File: src/cadence/mainnet/getNFTs.cdc [hunk 1]
```diff
@@ -3510,3 +3512,33 @@ pub fun getOzoneMetaverseNFT(owner: PublicAccount, id: UInt64): NFTData? {
...
+    return NFTData(
+        contract: contract,
+        id: nft!.id,
+        uuid: nft!.uuid,
+        title: nil,
+        description: nil,
+        external_domain_view_url: nil,
+        token_uri: nil,
+        media: [],
+        metadata: {}
```
Comment (niveda-krish) [inline src/cadence/mainnet/getNFTs.cdc:3542]: Do we have no metadata on-chain?

**Comment by niveda-krish** [comment]
Could you also resolve merge conflicts?

**Comment by javiermanzano** [comment]
> Could you also resolve merge conflicts?

Solved!

**Review by niveda-krish — APPROVED** [review]

**Comment by albeethekid** [comment]
@javiermanzano it looks like you've used a testnet address in src/cadence/mainnet/getNFTs.cdc

Can you please fix that up? And perhaps test to ensure your function is in working order?

**Comment by niveda-krish** [comment]
Hey @javiermanzano your testnet changes throw an error. I will be reverting the changes in our repo. Please send a  followup PR with the fix!

**Comment by javiermanzano** [comment]
hi @niveda-krish 

Sorry to be late for this but I was on holiday.

Unless I'm completely wrong, this is the mainnet contract -> https://flow-view-source.com/mainnet/account/0xa4e9020ad21eb30b/contract/SwaychainNFT

And I think that's the address that's on the PR. Could you point me the line where it's not like that? Probably I'm missing something

**Pull request merged**